packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -241,6 +242,16 @@ export namespace CommonCommands {
         category: VIEW_CATEGORY,
         label: 'Toggle Status Bar Visibility'
     });
+    export const PIN_TAB = Command.toDefaultLocalizedCommand({
+        id: 'workbench.action.pinEditor',
+        category: VIEW_CATEGORY,
+        label: 'Pin Tab'
+    });
+    export const UNPIN_TAB = Command.toDefaultLocalizedCommand({
+        id: 'workbench.action.unpinEditor',
+        category: VIEW_CATEGORY,
+        label: 'Unpin Tab'
+    });
```

Comment:
The eslint rule doesn't really work for `toDefaultLocalizedCommand` (I should fix this perhaps)

`Pin` and `Unpin` are the labels used by VSCode and can both be found in the `nls.metadata.json` file.

Reply:
I did wonder about whether the linting would catch it for the other NLS utilities. I'll update the text.

Reply:
I actually just noticed that VSCode is using `Pin editor`/`Unpin editor`. Since we do not only allow pinning on editors, we should maybe introduce new translations here? Or is `Pin editor` accurate enough for the command label?

Reply:
I think editors are going to be the most common targets, in any case, so I'll use '(un)pin editor' for now.